lib/generateBuffer.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const fs = require('fs');
+const schema = require('protocol-buffers-schema');
+const Pbf = require('pbf');
+const Compile = require('pbf/compile');
+
+const MVT_SPEC_VERSIONS = ['1.0.0', '1.0.1', '2.0', '2.1'];
+
+/**
+ * Generate a protocol buffer with a particular version of the Mapbox Vector Tile Specification or a custom .proto string
+ * @param {Object} json - the JSON representation of the protocol buffer, defined in /src
+ * @param {String} [proto] - an option parameter specifying which version of the specification to generate the buffer from
+ * OR a complete string representing a .proto file. If no string is provided, defaults to version 2.1
+ * @returns {Object} buffer - a protocol encoded buffere representing a valid or invalid Mapbox Vector Tile
+ */
+module.exports = function(json, proto) {
+  let proto_schema;
+  if (!proto) proto = '2.1';
```

Comment:
How about throwing if no `proto` is provided? That feels more robust to me since it would ensure each fixture self-documents which spec version it applies do.

Reply:
👍